provider/win_dump_utils.py

```diff
@@ -51,7 +51,8 @@ def generate_mem_dump(test, params, vm):
     output = vm.monitor.human_monitor_cmd('dump-guest-memory -w %s'
                                           % dump_file)
     if output:
-        test.fail("Save dump file failed as: %s" % output)
+        if "warning" not in output:
+            test.fail("Save dump file failed as: %s" % output)
```

Comment:
This seems too general to me, what is the warning exactly looks like? Can we distinguish it by the Exception type instead of a string filter?

Reply:
The waring looks like this:`warning: win-dump: number of QEMU CPUs is bigger than NumberProcessors (32) in guest Windows`.I think we can not distinguish it by a Exception type,it is a normal output.

Reply:
Okay, then:
```
if output and 'warning' not in output:
```

Reply:
Otherwise you won't go through below `else` steps when you have `warning`, that is not expected, right?

Reply:
Yes, exactly. Thanks!